### PR TITLE
EG: Added some code

### PR DESCRIPTION
**Pull request by TheEarlOfGray**

### CODE REVIEW

File: randomcode.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import random
+
+print(random.randint(1, 6))
```
Comment (gavthetallone) [inline randomcode.py:3]: Very random, I like it!

File: randomcode.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import random
```
Comment (gavthetallone) [inline randomcode.py:1]: Nicely imported, my friend.

**Review by gavthetallone — APPROVED** [review]
It just works!

File: randomcode.py (hunk 2 above)
Comment (TheEarlOfGray) [reply randomcode.py:1]: Thanks! This comment feature looks like it will be real useful during collaboration.

**Pull request merged**